packages/kit/src/views/Developer/pages/Gallery/Components/stories/TableGallery/components/data.ts

```
@@ -0,0 +1,58 @@
+    price: 95,
+    change24h: -2.1,
+  },
+];
+
+export const featureData: IFeatureData[] = [
+  { id: '1', feature: 'Custom Rendering', supported: 'Yes' },
+  { id: '2', feature: 'Sorting', supported: 'Yes' },
+  { id: '3', feature: 'Dragging', supported: 'Yes' },
+  { id: '4', feature: 'Skeleton Loading', supported: 'Yes' },
+  { id: '5', feature: 'Sticky Header', supported: 'Yes' },
+];
```

ESLint

[error] Insert `⏎`

(prettier/prettier)